docs/documentation/latex.md

```diff
@@ -2,7 +2,13 @@
 
 Makie can render LaTeX strings from the [LaTeXStrings.jl](https://github.com/stevengj/LaTeXStrings.jl) package using [MathTeXEngine.jl](https://github.com/Kolaru/MathTeXEngine.jl/).
 
-This engine supports a subset of LaTeX's most used commands, which are rendered quickly enough for responsive use in GLMakie.
+While this engine is responsive enough for use in GLMakie, it only supports a subset of LaTeX's most used commands.
+
+!!! note
+    Makie is able to render the subset of Latex strings supported by [MathTeXEngine.jl](https://github.com/Kolaru/MathTeXEngine.jl/).
+    Alternative scripts like ``\mathcal{X}`` or ``\mathbb{R}`` are not supported as of `MathTeXEngine.jl` `v0.5.1`.
+    In many cases, these characters can be replaced by unicode equivalents, e.g. ``ℝ`` for ``\mathcal{R}``.
+    A full list of Unicode replacements can be found [here](http://milde.users.sourceforge.net/LUCR/Math/unimathsymbols.pdf).
```

Comment:
only thing, I would not link with the word `here`, it's more user friendly to write out at least part of the url or a description